CHANGES.rst

```diff
@@ -113,6 +113,7 @@ Bugfixes
   in the title (:issue:`3998`)
 - Do not show pending registrants in public participant lists (:issue:`4017`)
 - Hide "External IDs" field if none are defined (:issue:`3857`)
+- Order Room Bookings (:issue:`4022`)
```

Comment:
I don't think anyone reading this changelog entry will know what it is about. I'd prefer this:

```rst
- Make list of event room bookings sortable (:issue:`4022`)
```